sleap_io/model/instance.py

```diff
@@ -0,0 +1,187 @@
+from __future__ import annotations
+from attrs import define, validators
+import attr
+from typing import List, Optional, Tuple, Union, Dict
+from sleap_io.model.video import Video
+from sleap_io.model.skeleton import Skeleton, Node
```

Comment:
Lets move these import to the root [__ init__.py](https://github.com/talmolab/sleap-io/blob/main/sleap_io/__init__.py) file and change these imports to
`from sleap import Video, Skeleton, Node`